tests/install_upgrade_operators/product_upgrade/utils.py

```diff
@@ -462,20 +464,16 @@ def verify_upgrade_ocp(
     )
 
 
-def get_all_cnv_alerts(prometheus, file_name, base_directory):
-    cnv_alerts = []
-    alerts_fired = prometheus.alerts()
-    for alert in alerts_fired["data"].get("alerts"):
-        if (
-            alert["labels"].get("kubernetes_operator_part_of")
-            and alert["labels"]["kubernetes_operator_part_of"] == "kubevirt"
-        ):
-            alert_name = alert["labels"]["alertname"]
-            if alert_name in WHITELIST_ALERTS_UPGRADE_LIST:
-                LOGGER.info(f"Whitelist alert {alert_name}")
-                continue
-            cnv_alerts.append(alert)
+def get_all_kubevirt_alerts(alert_dict: dict[str, Any]) -> list[dict[str, dict[str, str]]]:
+    return [
+        alert
+        for alert in alert_dict["data"].get("alerts")
+        if alert["labels"].get("kubernetes_operator_part_of") == "kubevirt"
+    ]
 
+
+def get_all_cnv_alerts(prometheus: Prometheus, file_name: str, base_directory: str) -> list[dict[str, dict[str, str]]]:
```

Comment:
So the difference between this one and `get_all_kubevirt_alerts` - is that `get_all_cnv_alerts` writes them to a file. Maybe it would be better to rename it?